tools/operator/testdata/kafka-couchdb-tls-network-spec.yml

```diff
@@ -22,6 +22,8 @@ metrics: true
 gossipEnable: false
 #! enable node ou's in fabric network (true/false)
 enableNodeOUs: true
+#! Provide kafka version
+kafkaVersion: 1.0.2
 
 #! Location where crypto-config, connection-profiles, channel-artifacts are stored
 artifactsLocation: /home/jenkins/gopath/src/github.com/hyperledger/fabric-test/fabric/internal/cryptogen/
```

Comment:
can we change this path to relative path instead of absolute path. Such that this file can be used in any system, with absolute path here, it may not be able to run when system does not have /home/jenkins